### PR TITLE
Feature/playwright migration

### PR DESCRIPTION
**Pull request by adam-buckley**
## Checklist
- [x] I'm using the correct PHP Version (8.1 for current, 7.4 for legacy).

## Description
Merging Playwright setup into develop

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,8 +8,6 @@
 /storage/session
 /log
 
-.phpunit.result.cache
```
Comment (DerekCrannaford) [inline .gitignore:11]: Why do we version this now?

File: test/playwright/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Get Cmfive Working
+
+- Use this codespace: https://github.com/2pisoftware/codespace_dev_box/tree/BoilerplateCore_Modules_Tests_Debug
```
Comment (DerekCrannaford) [inline test/playwright/README.md:3]: This now outdated? Also, per chatter elsewhere around doc's, this is not a cmfive public repo/reference?

**Review by DerekCrannaford — APPROVED** [review]
A couple of points raised, pedantic/confused?
Any response appreciated.

Found nothing to stop this going ahead.

File: test/playwright/README.md (hunk 2 above)
Comment (dragonflyfree) [reply test/playwright/README.md:3]: will remove the "Get Cmfive Working" part, was for Dane to follow while trying to get the newer codespace to be playwright friendly

**Pull request merged**